crates/rpc/rpc-eth-api/src/core.rs

```diff
@@ -540,6 +550,60 @@ where
             .await?)
     }
 
+    /// Handler for: `eth_getTransactionBySenderAndNonce`
+    async fn transaction_by_sender_and_nonce(
+        &self,
+        sender: Address,
+        nonce: U64,
+    ) -> RpcResult<Option<RpcTransaction<T::NetworkTypes>>> {
+        trace!(target: "rpc::eth", ?sender, ?nonce, "Serving eth_getTransactionBySenderAndNonce");
+        let nonce = nonce.saturating_to::<u64>();
```

Comment:
I believe this should be sufficient because U64 can only fit u64
```suggestion
        let nonce = nonce.to::<u64>();
```